docs/readthedocs/models/input/OLM.rst

```diff
@@ -0,0 +1,45 @@
+.. _olm_model:
+
+overhead line monitoring
+----
+Representation of an olm type.
+
+Attributes, Units and Remarks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Type Model
+""""""""""
+
++--------------+---------+---------------------------------------------+
+| Attribute    | Unit    | Remarks                                     |
++==============+=========+=============================================+
+| uuid         | --      |                                             |
++--------------+---------+---------------------------------------------+
+| id           | --      | Human readable identifier                   |
++--------------+---------+---------------------------------------------+
+| T_s          |  °C     | max. line temperature                       |
```

Comment:
Throughout the PowerSystemDataModel our variable naming shifted a bit towards expressiveness. It would be good, if you could consider some changes in the naming. E.g. `temperatureMax`.  Additionally, we have used camel case in the other documentation pages. So, it would be good, if we could stick with the same concept in all places.

Moreover, we have defined some [StandardUnits](https://github.com/ie3-institute/PowerSystemDataModel/blob/dev/src/main/java/edu/ie3/datamodel/models/StandardUnits.java). Maybe you could consider to switch the units to those.